paddle/fluid/inference/tensorrt/convert/test_activation_op.cc

```diff
@@ -23,7 +23,7 @@ namespace tensorrt {
 TEST(ReluOpConverter, main) {
   framework::Scope scope;
   std::unordered_set<std::string> parameters;
-  TRTConvertValidation validator(10, parameters, scope, 1000);
+  TRTConvertValidation validator(1, parameters, scope, 1000);
```

Comment:
Why 10 -> 1?

Reply:
10 represents the max batch size of the whole runing context.  Set 1 for edge testing.  Multi batch testing is not supported, and subsequent modifications are required.

Reply:
1 as an edge case is OK. but better to support `batch_size` 10 or other more than 1, be careful not to narrow the ability of the current implementation.